internal/runners/activate/activate.go

```diff
@@ -81,7 +62,12 @@ func (r *Activate) run(params *ActivateParams, activatorLoop activationLoopFunc)
 		}
 	}
 
-	go sendProjectIDToAnalytics(params.Namespace, filepath.Join(targetPath, constants.ConfigFileName))
+	// Send google analytics event with label set to project namespace
+	names, fail := project.ParseNamespaceOrConfigfile(params.Namespace.String(), filepath.Join(targetPath, constants.ConfigFileName))
+	if fail != nil {
+		logging.Debug("error resolving namespace: %v", fail.ToError())
```

Comment:
Should have some sort of fallback mechanic, currently it will still call `names.String()` which will cause a nil pointer panic.

Reply:
Thanks for catching that.